main.py

```diff
@@ -0,0 +1,7 @@
+import callhandle as ch
+print(ch.handle_call(["VIKING 11", "VIKING 12", "VIKING 13"], ["DARKSTAR", "PYTHON 11", "PYTHON 21", "VIKING 21", "VIKING 22"])) # One Element
```

Comment:
python code is generally set to some column width limit, e.g. 80, 100, or 120 (depends on yours, or your company/team's preference)

this line (and others) are too long. usually you want to add new lines to make things easier to read and fit on the screen. (even if you have a wide monitor, when you get into complex projects you will end up having 2-3 files open at the same time)

e.g.

```python
print(ch.handle_call(["VIKING 11", "VIKING 12", "VIKING 13"], ["DARKSTAR", "PYTHON 11", "PYTHON 21", "VIKING 21", "VIKING 22"])) # One Element
```

->

```python
print(
    ch.handle_call(
        ["VIKING 11", "VIKING 12", "VIKING 13"], 
        ["DARKSTAR", "PYTHON 11", "PYTHON 21", "VIKING 21", "VIKING 22"]
    )
) # One Element
```